api-samples/readingList/README.md

```diff
@@ -0,0 +1,13 @@
+# chrome.readingList API
+
+This sample demonstrates using the [chrome.readingList](https://developer.chrome.com/docs/extensions/reference/readingList/) API to view items in the reading list.
+
+## Overview
+
+Once this extension is installed, clicking this extension's action icon will open an extension page.
+
+<img src="screenshot.png" height=300 alt="Screenshot showing the chrome.readingList API demo running in Chrome.">
```

Comment:
```suggestion
## Overview

The extension's action icon opens a page that allows you to add new items, as well as update or delete existing ones.

<img src="screenshot.png" height=300 alt="Screenshot showing the chrome.readingList API demo running in Chrome.">

## Running this extension

1. Clone this repository.
2. Load this directory in Chrome as an [unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked).
3. Click on the action icon.
4. An extension page will open
```

It's a bit redundant, but I want to stick to the template we put together, in case a develoeprs lands here without knowing how to load an extension locally